docs/dev/design_documents/deprecations_in_resources.md

```diff
@@ -0,0 +1,68 @@
+# Deprecation Warnings Within Custom Resources
+
+In Chef 12, we introduced deprecation warnings within the chef-client. This allowed us to communicate future breaking changes to users. The warnings and integration within Test Kitchen have become a powerful tool, allowing users to future-proof their cookbooks and ease chef-client migration projects.
+
+This design extends the deprecation functionality to custom resources, allowing authors to warn consumers of future breaking changes to their resources.
+
+## Motivation
+
+    As an author of custom resources,
+    I want to warn users of my resources of future breaking changes to resources,
```

Comment:
There are two phrases in this line where "of" is repeated twice. This should be fixed for clarity, but I am unsure what words are intended to be used.

Reply:
I reworded this